lib/components/term-group.js

```diff
@@ -48,6 +48,7 @@ class TermGroup_ extends Component {
     const session = this.props.sessions[uid];
     const termRef = this.props.terms[uid];
     const props = getTermProps(uid, this.props, {
+      isSessionActive: uid === this.props.activeSession,
```

Comment:
Maybe it is better to have a isActive or isTermActive prop. A Term has no knowledge about what a Session is.